tarantool/mesh_connection.py

```diff
@@ -69,6 +100,10 @@ def format_error(address, err):
     if 'port' not in address or address['port'] is None:
         return format_error(address, 'port is not set or None')
 
+    for k, v in default_addr_opts.items():
+        if not k in address:
+            address[k] = v
```

Comment:
I've renamed `validate_address` to `prepare_address`.

It would be possible to split this function, but it is used in one meaning... That's why I didn't do it.